poms/bom/pom.xml

```diff
@@ -6266,6 +6266,17 @@
                 <groupId>io.quarkiverse.tika</groupId>
                 <artifactId>quarkus-tika</artifactId>
                 <version>${quarkiverse-tika.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>commons-logging</groupId>
+                        <artifactId>commons-logging</artifactId>
+                    </exclusion>
+                    <!-- It could be removed when https://github.com/quarkiverse/quarkus-tika/issues/109 gets resolved -->
```

Comment:
Unless we can get a quick release of `quarkus-tika` containing the fix for this, we should probably open a follow-up issue here as a reminder to remove this exclusion.

Reply:
I will take a new releas of `quarkus-tika` ASAP.

Reply:
It should be fine with `2.0.0.CR2`.